tests/test_config.c

```diff
@@ -463,6 +463,10 @@ void test_parse_webpa_url ()
 		addr_buf, 80, port_buf, 8), 0);
 	assert_string_equal (addr_buf, "mydns.mycom.net");
 	assert_string_equal (port_buf, "443");
+	assert_int_equal (parse_webpa_url ("https://mydns.mycom.net/api/v2/",
```

Comment:
I didn't even know this was an acceptable url.
But if so, we have to handle it.

Reply:
At our company we are seeing it. We are seeing a redirect that has an URI in it, and as we are using default HTTPS port, this bug was easily spotted.

I don't know if you @bill1600 or @schmidtw could merge it, if it also fits Comcast needs.
If it a reshape in any way, I'm all eyes and ears for your comments.